packages/rancher-istio/charts/1.6.400/configs/istio-base.yaml

```diff
@@ -0,0 +1,251 @@
+        {{- end }}
+        {{- if .Values.global.arch.ppc64le }}
+        ppc64le: {{ .Values.global.arch.ppc64le }}
+        {{- end }}
+        {{- if .Values.global.arch.s390x }}
+        s390x: {{ .Values.global.arch.s390x }}
+        {{- end }}
+      {{- if .Values.global.configValidation }}
+      configValidation: {{ .Values.global.configValidation }}
+      {{- end }}
+      {{- if .Values.global.controlPlaneSecurityEnabled }}
+      controlPlaneSecurityEnabled: {{ .Values.global.controlPlaneSecurityEnabled }}
+      {{- end }}
+      {{- if .Values.global.defaultNodeSelector }}
+      defaultNodeSelector: {{ .Values.global.defaultNodeSelector | toJson | indent 4 }}
+      {{- end }}
+      {{- if .Values.global.defaultPodDisruptionBudget.enabled }}
+      defaultPodDisruptionBudget:
+        enabled: {{ .Values.global.defaultPodDisruptionBudget.enabled }}
+      {{- end }}
+      defaultResources:
+        requests:
+          {{- if .Values.global.defaultResources.requests.cpu }}
+          cpu: {{ .Values.global.defaultResources.requests.cpu }}
+          {{- end }}
+      {{- if .Values.global.enableHelmTest }}
+      enableHelmTest: {{ .Values.global.enableHelmTest}}
+      {{- end }}
+      {{- if .Values.global.imagePullPolicy }}
+      imagePullPolicy: {{ .Values.global.imagePullPolicy }}
+      {{- end }}
+      imagePullSecrets: []
+      istioNamespace: istio-system
+      {{- if .Values.global.istiod.enabled }}
+      istiod:
+        enableAnalysis: {{ .Values.global.istiod.enableAnalysis }}
+        enabled: {{ .Values.global.istiod.enabled }}
+      {{- end }}
+      logging:
+        {{- if .Values.global.logging.level }}
+        level: "{{ .Values.global.logging.level }}"
+        {{- end }}
+      {{- if .Values.global.meshExpansion.enabled }}
+      meshExpansion:
+        enabled: {{ .Values.global.meshExpansion.enabled }}
+        useILB: {{ .Values.global.meshExpansion.useILB }}
+      {{- end }}
+      meshNetworks: {}
+      {{- if .Values.global.mountMtlsCerts }}
+      mountMtlsCerts: {{ .Values.global.mountMtlsCerts }}
+      {{- end }}
+      {{- if .Values.global.multiCluster.enabled }}
+      multiCluster:
+        clusterName: "{{ .Values.global.multiCluster.clusterName }}"
+        enabled: {{ .Values.global.multiCluster.enabled }}
+      {{- end }}
+      {{- if .Values.global.network }}
+      network: "{{ .Values.global.network }}"
+      {{- end }}
+      {{- if .Values.global.oneNamespace}}
+      oneNamespace: {{ .Values.global.oneNamespace }}
+      {{- end }}
+      {{- if .Values.global.operatorManageWebhooks }}
+      operatorManageWebhooks: {{ .Values.global.operatorManageWebhooks }}
+      {{- end }}
+      {{- if .Values.global.pilotCertProvider }}
+      pilotCertProvider: {{ .Values.global.pilotCertProvider }}
+      {{- end }}
+      {{- if .Values.global.priorityClassName }}
+      priorityClassName: {{ .Values.global.priorityClassName }}
+      {{- end }}
+      proxy:
```

Comment:
Think most things below would need conditionals and quotes, but thats ok for prev 1.